### PR TITLE
Issue triage: switch to our own `assign-by-label` action

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #16717 

I wrote my own "assign by label" action, that works slightly smarter than the one we were using previously: whenever a label was added, that one removed all assignees, and then tried to re-add them as appropriate for the current labels but the re-adding did not work well when multiple labels are specified. 
My action looks _which_ label was added or removed, and only assigns or unassigns the people who are specified for that label. It does not unassign other people that were assigned manually for example.

You can see it at https://github.com/cbjeukendrup/assign-by-label, and I have tested it at https://github.com/cbjeukendrup/MuseScore/issues/4. (edit: moved to https://github.com/musescore/github_actions/tree/main/assign-by-label)

### CODE REVIEW

File: .github/workflows/ci_issues_triage.yml [hunk 1]
```diff
@@ -19,41 +19,43 @@ jobs:
...
-    - uses: actions/checkout@v3
+  assign:
+    if: github.event.action == 'labeled' || github.event.action == 'unlabeled'
+    runs-on: ubuntu-latest
+    steps:
     - name: 'Set assignees on issue'
-      uses: kyanagimoto/assignee4label@main
+      uses: ./.github/actions/assign-by-label
       with:
-        github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```
Comment (cbjeukendrup) [inline .github/workflows/ci_issues_triage.yml:58]: Note to self: I should add this back

**Comment by igorkorsukov** [comment]
I would prefer this to be added to a separate repository in the MuseScore organization. What do you think of it? Create a repository? :)

**Comment by cbjeukendrup** [comment]
@igorkorsukov It was @Eism's idea to put it inside the main repo. However, I would be willing to transfer my repo from https://github.com/cbjeukendrup/assign-by-label to the `musescore` organisation.

**Comment by igorkorsukov** [comment]
> @igorkorsukov It was @Eism's idea to put it inside the main repo. However, I would be willing to transfer my repo from https://github.com/cbjeukendrup/assign-by-label to the `musescore` organisation.

I think the point here is that we need an action that does our needs. At the same time, needs can change, and people can change ..., so we need to make sure that we can change this action regardless of the person who created it if necessary (for example, if you go on a long trip :) ).

So, I will now add a new repository

**Comment by cbjeukendrup** [comment]
No, don't create a new repo, I'll transfer mine, I think that's easier

**Comment by cbjeukendrup** [comment]
Okay, I'll try to do it this way.

**Comment by cbjeukendrup** [comment]
Updated this PR; now https://github.com/musescore/github_actions/pull/1 needs to be merged first.

**Review by igorkorsukov — APPROVED** [review]

**Review by Eism — APPROVED** [review]

**Pull request merged**